deployment/.env.template

```diff
@@ -1,5 +1,16 @@
 # Make a copy called `.env` and replace all <PLACEHOLDERS>.
 
+# Use this to supply two compose files and override some options with the
+# settings from another compose file. Candidates are:
+# * docker-compose.latest.yml for using the latest dev version
+# * docker-compose.local.yml to run single services on a dev machine
+#COMPOSE_FILE=docker-compose.yml:docker-compose.latest.yml
+
+# Used as part of the hostnames: <service>.services-<DEPLOY_ENV>.raiden.network
+DEPLOY_ENV=dev
+# Prefix used for container names (default: current directory name)
+COMPOSE_PROJECT_NAME=dev
```

Comment:
How is the default the current directory name?

Reply:
If you don't set this variable name, docker compose takes the base name of the current directory.